cypress/integration/seededFlows/articleFlows/viewArticleDiscussion.spec.js

```diff
@@ -1,4 +1,4 @@
-describe('View article discussion', () => {
+describe.skip('View article discussion', () => {
```

Comment:
Instead of using `.skip` here, should I instead be using `.only` on the test that is _not_ flaky (shown below) so that instead of skipping the entire spec, I only run the passing test?

```
 it.only('initializes the follow button when a new comment is added', () => {
```